README.md

```diff
@@ -7,9 +7,9 @@
 
 Zoom images/videos on all your favorite websites (Facebook, Amazon, etc). Hover your mouse over any image on the [supported websites](https://github.com/extesy/hoverzoom/tree/master/plugins) and the extension will automatically enlarge the image to its full size, making sure that it still fits into the browser window.
 
-This is an open source version of the original HoverZoom extension, which is now overrun by malware and deleted from store. In this version all spyware has been removed, many bugs were fixed and new features were added. It doesn't collect any statistics whatsoever. The only permission it needs is to access data on all websites (to extract full images), and *optional* permissions to access browser history, download/save images or get tab urls for per-site configuration.
+This is an open-source version of the original HoverZoom extension, which is now overrun by malware and deleted from the store. In this version all spyware has been removed, many bugs were fixed and new features were added. It doesn't collect any statistics whatsoever. The only permission it needs is to access data on all websites (to extract full images), and *optional* permissions to access browser history, download/save images, or get tab URLs for per-site configuration.
 
-Sometimes sites change design, and when it happens, the extension needs to be updated. Please report any issues with zoom not working by filing an issue on https://github.com/extesy/hoverzoom/issues page.
+Sometimes sites change design, and when it happens, the extension needs to be updated. Please report any issues with Zoom not working by filing an issue on https://github.com/extesy/hoverzoom/issues page.
```

Comment:
Why is Zoom capitalized? It's a verb. Perhaps it should be updated to "with zooming not working" instead.